modules/client/examples/01-client/08-set-allowance.ts

```diff
@@ -14,32 +14,29 @@ This function updates the allowance approval to the amount specified.
 
 import {
   Client,
-  DaoDepositSteps,
-  UpdateAllowanceParams,
+  SetAllowanceParams,
```

Comment:
How can we import `SetAllowanceParams` here and have `const SetAllowanceParams` at line 25?